### ISSUE TITLE
Error when loading data

### ISSUE THREAD

**Issue by statzhero** [iangow/benchmark#1]
I get an error message when I try to load the data file as follows. Any suggestions? Thank you!

```
merged_voting_dt <- readRDS("merged_voting_dt.Rdata")
Error in readRDS("merged_voting_dt.Rdata") : unknown input format
```

And
```
load("merged_voting_dt.Rdata")
Error in load("merged_voting_dt.Rdata") : 
  bad restore file magic number (file may be corrupted) -- no data loaded
In addition: Warning message:
file ‘merged_voting_dt.Rdata’ has magic number 'versi'
  Use of save versions prior to 2 is deprecated 
```

Session
```
R version 4.0.3 (2020-10-10)
Platform: x86_64-apple-darwin17.0 (64-bit)
Running under: macOS Big Sur 10.16

Matrix products: default
LAPACK: /Library/Frameworks/R.framework/Versions/4.0/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods  
[7] base     

other attached packages:
[1] lfe_2.8-6    Matrix_1.3-2 xtable_1.8-4 tidyr_1.1.3  dplyr_1.0.5 

loaded via a namespace (and not attached):
 [1] Formula_1.2-4    magrittr_2.0.1   tidyselect_1.1.0
 [4] lattice_0.20-41  R6_2.5.0         rlang_0.4.10    
 [7] tools_4.0.3      parallel_4.0.3   grid_4.0.3      
[10] DBI_1.1.1        ellipsis_0.3.1   yaml_2.2.1      
[13] assertthat_0.2.1 tibble_3.0.6     lifecycle_1.0.0 
[16] crayon_1.4.1     purrr_0.3.4      vctrs_0.3.6     
[19] glue_1.4.2       sandwich_3.0-0   compiler_4.0.3  
[22] pillar_1.4.7     generics_0.1.0   pkgconfig_2.0.3 
[25] zoo_1.8-8       
```

**Comment by iangow** [comment]
The data are stored using Git LFS. Have you used `git lfs fetch` or equivalent to get the full file?

**Comment by iangow** [comment]
The file should be about 28.4 MB:

```
(base) iangow@Ians-Mac-mini benchmark % ls -hlt
total 58224
-rw-r--r--   1 iangow  staff   204B 17 Mar 14:32 benchmark.Rproj
-rw-r--r--   1 iangow  staff   783B  6 Mar 08:54 README.md
-rw-rw-rw-   1 iangow  staff   5.4K  5 Mar 11:28 benchmark.R
-rw-r--r--   1 iangow  staff    37K  4 Mar 17:09 benchmark.tex
drwxrwxrwx  27 iangow  staff   864B  4 Mar 17:09 cache
-rw-r--r--   1 iangow  staff    28M  4 Mar 15:47 merged_voting_dt.Rdata
```

**Comment by statzhero** [comment]
Woops, this must be the issue. Will try LFS.

**Comment by statzhero** [comment]
All good. :+1:

Mac mini (2018)
3 GHz 6-Core Intel Core i5
32 GB 2667 MHz DDR4

```
> system.time(source("benchmark.R", echo = FALSE))
   user  system elapsed 
 86.443   3.552  88.415 
```

**Comment by iangow** [comment]
Good to hear. This is just a pretty random "real world" benchmark I made (took some actual code I used in a paper that I know was a little slow for my co-author running on an older MBP and streamlined it). Another [benchmark](https://gist.github.com/iangow/fa3637e2d7df22cac57174a0415374d2) uses a Python notebook from a book and gives very similar results. And I seem similar comparative performance in longer-running tasks involving common-for-me read-data-from-disk-process-write-to-disk workflows.
